sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -247,33 +331,22 @@ def _make_metric_list_to_collect(self, custom_metrics):
         # If several check instances are querying the same server host, it can be wise to turn these off
         # to avoid sending duplicate metrics
         if is_affirmative(self.instance.get('include_instance_metrics', True)):
-            for name, counter_name, instance_name in self.INSTANCE_METRICS:
-                try:
-                    sql_type, base_name = self.get_sql_type(counter_name)
-                    cfg = {
-                        'name': name,
-                        'counter_name': counter_name,
-                        'instance_name': instance_name,
-                        'tags': tags,
-                    }
+            self._add_performance_counters(
+                chain(self.INSTANCE_METRICS, self.INSTANCE_METRICS_TOTAL), metrics_to_collect, tags, db=None
+            )
 
-                    metrics_to_collect.append(
-                        self.typed_metric(
-                            cfg_inst=cfg, table=DEFAULT_PERFORMANCE_TABLE, base_name=base_name, sql_type=sql_type
-                        )
-                    )
-                except SQLConnectionError:
-                    raise
-                except Exception:
-                    self.log.warning("Can't load the metric %s, ignoring", name, exc_info=True)
-                    continue
+        # populated through autodiscovery
+        if self.databases:
+            for db in self.databases:
+                self._add_performance_counters(self.INSTANCE_METRICS_TOTAL, metrics_to_collect, tags, db=db)
```

Comment:
```suggestion
            if self.databases:
                for db in self.databases:
                    self._add_performance_counters(self.INSTANCE_METRICS_TOTAL, metrics_to_collect, tags, db=db)
```

Should these instance metrics only be included if the option is enabled?

Reply:
Also for instance-level metrics, wouldn't the database not matter?

Reply:
These were split off from the previous `INSTANCE_METRICS` because they are ones that can be explicitly tied to databases.  The extra `db=db` kwarg will transform them from instance-level to database-level.